botocore/handlers.py

```diff
@@ -300,6 +300,17 @@ def copy_snapshot_encrypted(params, request_signer, **kwargs):
     params['PresignedUrl'] = presigned_url
 
 
+def json_encode_policies(params, model, **kwargs):
+    input_shape = model.input_shape
+    if input_shape is not None:
+        for member_name, member_shape in input_shape.members.items():
+            param = params.get(member_name)
+            if member_shape.name == 'policyDocumentType' and \
+                    param is not None and \
```

Comment:
Is the check of if it is ``None`` needed because you are already checking it it an instance of dict?